### PR TITLE
clean up source code

### PR DESCRIPTION
**Pull request by bkamins**
Changes:
* vast majority: removing spaces around `=` in kwargs
* added test for deprecated functionality of sort (I will mark it in the source code)

### CODE REVIEW

File: test/deprecated.jl [hunk 1]
```diff
@@ -140,4 +140,13 @@ end
     @test df == DataFrame(a=[1, 3, 4], b=1, c=2)
 end
 
+@testset "deprecated sort" begin
```
Comment (bkamins) [inline test/deprecated.jl:143]: @nalimilan - these are the tests of deprecated functionality

**Review by nalimilan — APPROVED** [review]

File: src/abstractdataframe/abstractdataframe.jl [hunk 2]
```diff
@@ -1038,11 +1038,11 @@ julia> filter(row -> row.x > 1, df)
 
 julia> filter(row -> row["x"] > 1, df)
 2×2 DataFrame
- Row │ x      y      
+ Row │ x      y
      │ Int64  String
 ─────┼───────────────
    1 │     3  b
-   2 │     2  a                                                                    
+   2 │     2  a
```
Comment (bkamins) [inline src/abstractdataframe/abstractdataframe.jl:1045]: @nathanrboyer - for your future PRs can you please make sure to set your editor to trim trailing spaces? Thank you! (here is an example of a fix to your recent PR)

File: test/string.jl [hunk 3]
```diff
@@ -52,22 +52,22 @@ end
...
-    wide = DataFrame(id = 1:12,
-                     a  = repeat([1:3;], inner = [4]),
-                     b  = repeat([1:4;], inner = [3]),
-                     c  = randn(12),
-                     d  = randn(12))
+    wide = DataFrame(id=1:12,
+                     a =repeat([1:3;], inner=[4]),
+                     b =repeat([1:4;], inner=[3]),
+                     c =randn(12),
+                     d =randn(12))
```
Comment (bkamins) [inline test/string.jl:70]:
```suggestion
                     a=repeat([1:3;], inner=[4]),
                     b=repeat([1:4;], inner=[3]),
                     c=randn(12),
                     d=randn(12))
```

**Comment by bkamins** [comment]
Thank you!

File: src/abstractdataframe/abstractdataframe.jl (hunk 2 above)
Comment (nathanrboyer) [reply src/abstractdataframe/abstractdataframe.jl:1045]: Okay, I see.

**Pull request merged**